examples/morpheusvm/tests/integration/integration_test.go

```diff
@@ -221,7 +221,13 @@ var _ = ginkgo.BeforeSuite(func() {
 			genesisBytes,
 			nil,
 			[]byte(
-				`{"parallelism":3, "testMode":true, "logLevel":"debug"}`,
+				`{
+				  "config": {
+				    "parallelism":3,
```

Comment:
Noticed while reviewing this PR, it looks like `parallelism` field is unused in both this and TokenVM. Could we remove it in either this or a follow up?

Reply:
Yeah looks stale. Will create a follow-up.

Reply:
Followup PR: https://github.com/ava-labs/hypersdk/pull/1158